lib/sidekiq_bus/version.rb

```diff
@@ -1,3 +1,3 @@
 module SidekiqBus
-  VERSION = "0.6.3"
+  VERSION = "0.6.4"
```

Comment:
I'd rather add the changes to a changelog and release the gem versions in a separate commit/tag.

Reply:
fair enough